sdk/purview/purview-workflow-rest/samples-dev/submitUserRequestsSample.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import createPurviewWorkflowClient, {
+  PurviewWorkflowClient,
+  SubmitUserRequestsParameters,
+  isUnexpected,
+} from "@azure-rest/purview-workflow";
+import { UsernamePasswordCredential } from "@azure/identity";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+/**
+ * This sample demonstrates how to Submit a user request for requestor, a user  request describes user ask to do operation(s) on Purview. If any workflow's trigger matches with an operation in request, a run of the workflow is created. Before user submit a user request, a workflow should be created.
+ *
+ * @summary Submit a user request for requestor, a user  request describes user ask to do operation(s) on Purview. If any workflow's trigger matches with an operation in request, a run of the workflow is created.
+ * x-ms-original-file: specification/purview/data-plane/Azure.Analytics.Purview.Workflow/preview/2022-05-01-preview/examples/SubmitUserRequests.json
+ */
+
+async function userRequestsSubmit(
+  client: PurviewWorkflowClient,
+  userRequest: SubmitUserRequestsParameters
+) {
+  const result = await client.path("/userrequests").post(userRequest);
+  if (isUnexpected(result)) {
+    throw result.body.error;
+  }
+  console.log(`The submitted user request is ${result.body}`);
```

Comment:
I would recommend putting this inline in `main` instead of on its own function

Reply:
Thanks for your suggestion, I'd like keep main() function simple to make user could see the flow clearly, so I prefer to putting the single API in an individual function.

Reply:
We have to keep in mind that these are samples that our customers are going to be looking at to learn how to use the library and oftentimes they copy/paste the code in their applications.

I  see the following issues with the operation call into its own function:

1. It adds cognitive overload, making users go back and forth between the main and `userRequestsSubmit`. There is no complex logic in the abstracted function so the improvement in readability is marginal and IMHO not worth it for the samples. 
2. It may also cause confusion to the users, thinking they could import `userRequestsSubmit` from the package.
3. It is not clear from looking at the main what the outcome of the sample is, you have to go to the function to see that the result is used to log. At the very least the log statement should be in the main.

Reply:
Hi @joheredi, SubmitUserRequestsSamples is a particular case that this scenario just involve one API, that's why it has only one API in this sample file, for this sample, I agree your suggestion, I have done  the changes, thanks.